src/topology/metric_space/baire.lean

```diff
@@ -86,18 +90,27 @@ end
 
 end is_Gδ
 
+/-- A set `s` is called *residual* if it includes a dense `Gδ` set. If `α` is a Baire space
+(e.g., a complete metric space), then residual sets form a filter. Though we define this filter
```

Comment:
The sentence is strange: residual sets always form a filter, you don't need a Baire space.

Reply:
Do you mean residual sets in the sense of the definition or in the sense of the comment and `mem_residual`?

Reply:
Ah, nevermind, the intersection of two residual sets is not residual in general. Yes, it is disturbing that you define a filter which does in fact *not* coincide with the definition `includes a dense Gdelta set` in general. Could you expand the docstring to help stupid readers? :-)

Reply:
Done. Two days ago I was a stupid author who tried to define `residual` as the filter of dense Gδ sets in any topological space.